assets/component-menu-drawer.css

```
@@ -215,11 +215,27 @@ details[open].menu-opening > .menu-drawer__submenu {
 }
 
 .menu-drawer__utility-links {
-  padding: 2rem;
+  padding: 0;
   background-color: rgba(var(--color-foreground), 0.03);
   position: relative;
 }
 
+.menu-drawer__utility-links--social-media-links {
+  padding: 2rem;
+}
+
+@media screen and (max-width: 749px) {
+  .menu-drawer__utility-links--account {
+    padding: 2rem;
+  }
+}
+
+@media screen and (max-width: 989px) {
+  .menu-drawer__utility-links--localizations {
```

Maybe too much, but this prevents repeating the same styling 3 times 🤷 
```suggestion
  .menu-drawer__utility-links--localizations:where(:not(.menu-drawer__utility-links--social-media-links)) {
```